enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/api/APIFactory.java

```diff
@@ -91,8 +101,32 @@ public void removeApi(API api) {
         apis.remove(apiKey);
     }
 
+    /**
+     * Validate the GraphQL schema before adding the API.
+     *
+     * @param api api details
+     * @return whether the api is an valid graphQL API.
+     */
+    private boolean isValidGraphQLAPI(Api api) {
+        if (!api.getGraphQLSchema().isEmpty()) {
+            String sdl = api.getGraphQLSchema();
+            SchemaParser schemaParser = new SchemaParser();
+            try {
+                schemaParser.parse(sdl);
+                return true;
+            } catch (SchemaProblem exception) {
+                logger.error("Error while parsing graphQL sdl for API : {} version : {}", api.getTitle(),
```

Comment:
shall we print apiUUID too? It is easy to query the logs in that way.

Reply:
actually we can remove this block now as adapter does this parsing. @slahirucd7 right?

Reply:
Validation is already done at adapter level. Hence schema validation in enforcer is removed.